src/trackExtensions.ts

```diff
@@ -0,0 +1,62 @@
+import { Extension, getExtensionHash, getLogs } from '@colony/colony-js';
+import { extensionSpecificEventsListener } from './eventListener';
+
+import networkClient from './networkClient';
+import { verbose } from './utils';
+
+const supportedExtensionIds = [getExtensionHash(Extension.VotingReputation)];
```

Comment:
Should oneTxPayment be here also?

Reply:
OneTxPayment is enabled by default so we don't have to worry about its Initialised events

Reply:
This is determined by the lack of `initializationParams` in a particular extension config in CDapp's `constants/extensions.ts`. I believe the `OneTxPaymentInitialised` event is still possible to be emitted, but we can ignore it

Reply:
Maybe leave a comment about this ?

This is kind of a _hidden_ function of the contracts, so future devs that come across this will not fall into traps.